jupyterhub/handlers/base.py

```diff
@@ -994,6 +1003,11 @@ def abort():
                 SERVER_STOP_DURATION_SECONDS.labels(
                     status=ServerStopStatus.success
                 ).observe(toc - tic)
+                self.event_log.emit('hub.jupyter.org/server-action', 1, {
+                        'action': 'stop',
+                        'username': user.name,
+                        'servername': server_name
```

Comment:
Should these include durations like the prometheus metrics?

Reply:
Is it worth including the requesting user in these events (not always the same as the server owner)?

Reply:
Maybe! I want to keep the first version of these as simple as possible, though - since the focus is on the eventlogging infrastructure itself.. We can easily add more later on. Is that ok?